borgmatic/hooks/pushover.py

```diff
@@ -0,0 +1,89 @@
+import logging
+
+import requests
+
+logger = logging.getLogger(__name__)
+
+
+def initialize_monitor(
+    ping_url, config, config_filename, monitoring_log_level, dry_run
+):  # pragma: no cover
+    '''
+    No initialization is necessary for this monitor.
+    '''
+    pass
+
+
+def ping_monitor(hook_config, config, config_filename, state, monitoring_log_level, dry_run):
+    '''
+    Post a message to the configured Pushover application.
+    If this is a dry run, then don't actually update anything.
+    '''
+
+    run_states = hook_config.get('states', ['fail'])
+
+    if state.name.lower() not in run_states:
+        return
+
+    dry_run_label = ' (dry run; not actually updating)' if dry_run else ''
+
+    state_config = hook_config.get(
+        state.name.lower(),
+        {
+            'message': state.name.lower(),
```

Comment:
This seems unnecessary given that you also default `message` below...?

Reply:
agreed: https://github.com/borgmatic-collective/borgmatic/commit/46ebb0cebbebc691d734e3f850a2ee1b7d480c68

Reply:
I've added to the above commit to fix it breaking a few tests: https://github.com/borgmatic-collective/borgmatic/commit/ef66349674c10fec1b84ead129a4028c98340bb6